.github/workflows/continuous-integration-karma.yml

```diff
@@ -38,8 +38,11 @@ jobs:
         env:
           CI: true
 
-      - name: Run integration tests
-        run: npm run test:karma -- --headless --snapshots --viewport=1600:1000
+      - name: Run edit-story integration tests
```

Comment:
IMHO, we should provide a separate `continuous-integration-karma-edit-story.yml` and `continuous-integration-karma-dashboard.yml`. They can run in parallel and the parameters between them are like to split more as well.